src/Data/Subscription/SubscriptionDataClass.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  ClientError,
+  getInstanceId,
+  provideDataClass,
+  unstable_JrRestApi,
+} from 'scrivito'
+
+interface Topic {
+  id: string
+  title: string
+  description?: string
+}
+
+export const Subscription = provideDataClass('Subscription', {
+  connection: {
+    async index() {
+      const myTopics = (
+        sanitizeResults(
+          await unstable_JrRestApi.fetch(
+            `neoletter/instances/${getInstanceId()}/my/topics`,
+          ),
+        ) as { results: Topic[] }
+      ).results
+
+      const mySubscribedTopicIds = await fetchMySubscribedTopicIds()
+
+      return {
+        results: myTopics.map(({ id, description, title }) => ({
+          id,
+          isConsentGiven: mySubscribedTopicIds.includes(id),
```

Comment:
If we don't convert `status` to a boolean, but to `status` `"revoked"`/`"given"` instead, we would have not need to implement boolean handling into `DataIconWidget` etc.

Reply:
From the perspective of the end-user there is no big different between unknown and revoked. How about we introduce the artificial status `"given"`/`"notGiven"` to avoid the word "revoke"?

Reply:
Or `"subscribed"`/`"unsubscribed"`.